util/test/chpl_launchcmd.py

```diff
@@ -558,7 +558,7 @@ class OurFormatter(argparse.ArgumentDefaultsHelpFormatter, argparse.RawDescripti
         parser = argparse.ArgumentParser(
             description=__doc__,
             formatter_class=OurFormatter)
-        parser.add_argument('-v', '--verbose', action='store_true',
+        parser.add_argument('--CHPL_LAUNCHCMD_DEBUG', action='store_true',
```

Comment:
I forgot about this, but this breaks things trying to access `args.verbose` and `args.hostlist` in this script. The `dest` argument to add_argument() should fix it. ie

```python
parser.add_argument('--CHPL_LAUNCHCMD_DEBUG', action='store_true', dest='verbose',

... and ...

parser.add_argument('--CHPL_LAUNCHCMD_HOSTLIST', dest='hostlist',
```

Sorry I missed that yesterday!